BookingsApi/BookingsApi.UnitTests/Services/SingledayHearingAsynchronousProcessTests.cs

```diff
@@ -52,5 +52,30 @@ public async Task Should_publish_messages_for_single_day_booking()
             messages.Count(x => x.IntegrationEvent is HearingIsReadyForVideoIntegrationEvent).Should().Be(createConfereceMessageCount);
             messages.Count(x => x.IntegrationEvent is ExistingParticipantHearingConfirmationEvent).Should().Be(hearingConfirmationForExistingParticipantsMessageCount);
         }
+        
+        [Test]
+        public async Task Should_publish_messages_for_single_day_booking_with_panel_member_and_ejud_new_template_off()
+        {
+            var hearing = new VideoHearingBuilder(addJudge:false).WithCase().WithJudiciaryPanelMember().WithJudiciaryJudge().Build();
+            hearing.Participants[0].Person.GetType().GetProperty("CreatedDate").SetValue(hearing.Participants[0].Person, 
+                hearing.Participants[0].Person.CreatedDate.AddDays(-10), null);
+           
+            ((FeatureTogglesStub)_featureToggles).NewTemplates = false;
+
+            var createConfereceMessageCount = 1;
+            var newParticipantWelcomeMessageCount = hearing.Participants.Count(x => x is not JudicialOfficeHolder && x is not Judge);
+            var judiciaryPersons = hearing.JudiciaryParticipants.Count;
```

Comment:
can probably just inline this below, instead of having a separate variable

`hearing.JudiciaryParticipants.Count + hearing.Participants.Count`

Reply:
done